test/cookies/manifest.json

```diff
@@ -0,0 +1,120 @@
+{
+    "http_server": {
+        "addr": ":9999",
+        "dir": "../_res",
+        "testmode": false
+    },
+    "name": "check cookies from server as defined in test request",
+    "tests": [
+        {
+            "name": "set a sess cookie",
+            "request": {
+                "server_url": "http://localhost:9999",
+                "endpoint": "bounce-json",
+                "method": "POST",
+                "set_cookies": [
+                    {
+                        "name": "sess",
+                        "value": "my_sess"
+                    },
+                    {
+                        "name": "dummy",
+                        "value": "whatever",
+                        "path": "/mypath"
+                    }
+                ],
+                "body": {}
+            },
+            "response": {
+                "statuscode": 200,
+                "header": {
+                    "Set-Cookie": [
+                        "sess=my_sess",
+                        "dummy=whatever; Path=/mypath"
```

Comment:
We need a more explicit way to test cookies.

// Match the Cookie JAR here
`"cookie": {

}`